### ISSUE TITLE
As

### ISSUE THREAD

**Issue by fadol83-lab** [fadol83-lab/2FA#2]
<p>Issue details: Open <a href="https://accessibility.browserstack.com/site-scanner/projects/github.com_FBichAd_9964283_a11y/builds?thBuildId=dazxyjjl1w6vsbmwisynuuiohq0nq3vapl6ufrv4&recurring=true&tunnel_vision=false&minimal=true&id=github.com_FBichAd_9964283_a11y&tab=ALLISSUES&activeTab=issue-type&activeReviewStatus=needs-review&activeViolationId=17726&activeComponentId=span%23prc-Button-Label-pTQ3x&activeBehaviorValue=&activeIssueId=542442205">URL</a> on Browserstack</p><br />Issue details<table><tr><th>Item&nbsp;&nbsp;</th><th>Data&nbsp;&nbsp;</th></tr><tr><td>Issue name&nbsp;&nbsp;</td><td>Elements must meet minimum color contrast ratio thresholds&nbsp;&nbsp;</td></tr><tr><td>Issue description&nbsp;&nbsp;</td><td>Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds&nbsp;&nbsp;</td></tr><tr><td>More Info on issue&nbsp;&nbsp;</td><td><a href="https://accessibility.browserstack.com/api/more-info/a11y-engine-5.7/color-contrast">https://accessibility.browserstack.com/api/more-info/a11y-engine-5.7/color-contrast</a>&nbsp;&nbsp;</td></tr><tr><td>Affected component&nbsp;&nbsp;</td><td>span.prc-Button-Label-pTQ3x&nbsp;&nbsp;</td></tr><tr><td>WCAG tags&nbsp;&nbsp;</td><td>WCAG 1.4.3&nbsp;&nbsp;</td></tr><tr><td>Best Practice issue&nbsp;&nbsp;</td><td>No&nbsp;&nbsp;</td></tr><tr><td>Experimental&nbsp;&nbsp;</td><td>No&nbsp;&nbsp;</td></tr><tr><td>Needs review&nbsp;&nbsp;</td><td>Yes&nbsp;&nbsp;</td></tr><tr><td>Impact&nbsp;&nbsp;</td><td>Serious&nbsp;&nbsp;</td></tr><tr><td>Affected page&nbsp;&nbsp;</td><td><a href="https://github.com/mdn/browser-compat-data/blob/main/http%2Fmethods.json">https://github.com/mdn/browser-compat-data/blob/main/http%2Fmethods.json</a>&nbsp;&nbsp;</td></tr><tr><td>How to fix&nbsp;&nbsp;</td><td>Get more details here<a href="https://accessibility.browserstack.com/site-scanner/projects/github.com_FBichAd_9964283_a11y/builds?thBuildId=dazxyjjl1w6vsbmwisynuuiohq0nq3vapl6ufrv4&recurring=true&tunnel_vision=false&minimal=true&id=github.com_FBichAd_9964283_a11y&tab=ALLISSUES&activeTab=issue-type&activeReviewStatus=needs-review&activeViolationId=17726&activeComponentId=span%23prc-Button-Label-pTQ3x&activeBehaviorValue=&activeIssueId=542442205">(https://accessibility.browserstack.com/site-scanner/projects/github.com_FBichAd_9964283_a11y/builds?thBuildId=dazxyjjl1w6vsbmwisynuuiohq0nq3vapl6ufrv4&amp;recurring=true&amp;tunnel_vision=false&amp;minimal=true&amp;id=github.com_FBichAd_9964283_a11y&amp;tab=ALLISSUES&amp;activeTab=issue-type&amp;activeReviewStatus=needs-review&amp;activeViolationId=17726&amp;activeComponentId=span%23prc-Button-Label-pTQ3x&amp;activeBehaviorValue=&amp;activeIssueId=542442205)</a>&nbsp;&nbsp;</td></tr></table>
